ci/gpu/build.sh

```diff
@@ -46,7 +46,7 @@ conda install "rmm=$MINOR_VERSION.*" "nvstrings=$MINOR_VERSION.*" "cudatoolkit=$
               "dask>=2.1.0" "distributed>=2.1.0" "numpy>=1.16" "double-conversion" \
               "rapidjson" "flatbuffers" "boost-cpp" "fsspec>=0.3.3" "dlpack" \
               "feather-format" "cupy>=6.0.0" "arrow-cpp=0.14.1" "pyarrow=0.14.1" \
-              "pandas>=0.24.2,<0.25"
+              "fastavro>=0.22.0" "pandas>=0.24.2,<0.25"
```

Comment:
We should make a pr to https://github.com/rapidsai/gpuci-build-environment adding fastavro to the build container instead of doing it here. I'll handle.

Reply:
Added to PR here: https://github.com/rapidsai/gpuci-build-environment/pull/35

Reply:
Sounds good to me.

Reply:
This is fine for now and we'll follow up with a separate PR removing it from here once the above PR is merged.